sql/updates/world/2015_09_14_00_world.sql

```diff
@@ -0,0 +1,64 @@
+-- creature (2 entries)
+SET @CGUID = 1000000;
+-- gameobject (2 entries)
+SET @OGUID = 1000000;
+
+-- magnetic pull
+delete from `spell_scripts` where `id`=54517;
+delete from `spell_script_names` where `spell_id`=54517;
+insert into `spell_script_names` (`spell_id`,`scriptname`) values (54517,"spell_thaddius_magnetic_pull");
+
+-- feugen/stalagg
+update `creature_template` set `ScriptName`="npc_stalagg" where `entry`=15929;
+update `creature_template` set `ScriptName`="npc_feugen" where `entry`=15930;
```

Comment:
The update commands don't do any harm, they are just redundant if the names are there already.